src/components/DefaultSectionContainer/DefaultSectionContainer.tsx

```diff
@@ -1,24 +1,33 @@
 import type { StyleProps } from '@chakra-ui/react';
 import React from 'react';
 import { Box } from '@chakra-ui/react';
+import convertToKebabCase from '../../utils/convertToKebabCase';
 
 export interface DefaultSectionContainerProps extends StyleProps {
   children: JSX.Element | JSX.Element[];
+  title?: string;
 }
 
 export const DefaultSectionContainer: React.FC<DefaultSectionContainerProps> = ({
   children,
   backgroundColor,
+  title,
   ...props
-}: DefaultSectionContainerProps) => (
-  <Box
-    position="relative"
-    paddingY={backgroundColor ? ['12', null, '28'] : 'unset'}
-    marginY={!backgroundColor ? ['8', null, '28'] : 'unset'}
-    overflow={backgroundColor ? 'hidden' : 'unset'}
-    backgroundColor={backgroundColor}
-    {...props}
-  >
-    {children}
-  </Box>
-);
+}: DefaultSectionContainerProps) => {
+  const kebabCaseTitle = title ? convertToKebabCase(title) : '';
```

Comment:
A empty string is a invalid value because the id attribute must be at least one character https://html.spec.whatwg.org/multipage/dom.html#global-attributes:the-id-attribute-2
```suggestion
  const kebabCaseTitle = title ? convertToKebabCase(title) : undefined;
```